RELEASES.md

```diff
@@ -784,6 +784,7 @@ Language
 - [You can now use `#[repr(transparent)]` on univariant `enum`s.][68122] Meaning
   that you can create an enum that has the exact layout and ABI of the type
   it contains.
+- [It is no longer disallowed to apply an attribute procedural macro to a module.][77098]
```

Comment:
You should also add the link to PR here:
https://github.com/rust-lang/rust/blob/4c0c5e099a3b1f1c6ad53115189c2710495588b3/RELEASES.md#L843-L854

Reply:
Ah, I mean, you should add a link to PR that disallowed the behavior, not this PR (and issue number is mismatch).

Reply:
@JohnTitor Oh, that makes much more sense! Will do now